src/pages/workspaces/workspace/submissionHistory/SubmissionDetails.js

```diff
@@ -424,6 +424,7 @@ const SubmissionDetails = _.flow(
    */
   const {
     cost,
+    perWorkflowCostCap,
```

Comment:
Where does this value come from?  I'm not seeing it show up

Reply:
Right but where does that come from?  How does it get populated?

Reply:
I think from rawls, there is this PR in review that adds it to the model: https://github.com/broadinstitute/rawls/pull/3191

Reply:
OH so it's not even available yet?!  No wonder!